sdks/java/container/java25/option-java25-security.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "java-security",
+  "enabled": true,
+  "options": {
+    "properties": {
+      "java.security.properties": "/opt/apache/beam/options/java21-security.properties"
```

Comment:
Should this point to java25-security.properties?

Reply:
good catch, fixed